ddp-workspace/projects/ddp-sdk/src/lib/directives/invitationCodeFormatter.directive.ts

```diff
@@ -0,0 +1,98 @@
+import { Directive, ElementRef, forwardRef, HostListener, Input } from '@angular/core';
+import { MAT_INPUT_VALUE_ACCESSOR } from '@angular/material';
+import { ControlValueAccessor, NG_VALUE_ACCESSOR } from '@angular/forms';
+import { InvitationCodeInputFormatter } from '../utility/invitationCode/invitationCodeInputFormatter';
+
```

Comment:
Rewrote directive delegating all the formatting and positioning of cursor to invitationCodeInputFormtter